framework/src/test/java/org/checkerframework/framework/test/junit/ValueTest.java

```diff
@@ -26,7 +26,9 @@ public ValueTest(List<File> testFiles) {
                 "value",
                 // Ignore the test suite's usage of qualifiers in illegal locations.
                 "-AignoreTargetLocations",
-                "-Astubs=tests/value/minints-stub.astub:tests/value/lowercase.astub",
+                ("-Astubs=tests/value/minints-stub.astub").replace("/", File.separator)
```

Comment:
Can this use `TestUtilities.adapt` or would that be a bad dependency from `framework` to `framework-test`?
I wonder whether `adapt` should also replace `:` and here you could just write `TestUtilities.adapt("tests/value/minints-stub.astub:tests/value/lowercase.astub")`?